### DISCUSSION TITLE
What is pose truth value? How to determine millimeter-level positioning

### DISCUSSION THREAD

**Discussion by Carloslius** [jyjblrd/Mocap-Drones#31] [Q&A]
What is pose truth value? How to determine millimeter-level positioning

**Comment by jyjblrd** [comment]
What do you mean "pose truth value"? We don't actually know the ground truth of the camera poses, we simply estimate them to a high level of accuracy using the 7 point algorithm and bundle adjustment.

**Reply by Carloslius** [reply]
Hello, sorry to bother you. What I want to ask is how to determine the accuracy of millimeter positioning? Is it by comparing the preset point pose of the drone with the pose calculated by the 7 point algorithm and bundle adjustment? Thank you

**Reply by 0i0i0i** [reply]
can you explain a little bit about this 7 point and bundle adjustment algorithm? does this operation update the camera poses only or it also estimate the error and reduce if of the ir leds on drones? During this operation, does it need 7 ir leds whose positiions in the world coorinate system are already known?
